<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">PicnicHat Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Buttons</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Buttons</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="7" column="2">
              <widget class="QPushButton" name="pushButton_buttons_apply">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEdit_buttons_power"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>POWER</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Long push:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="lineEdit_buttons_reset"/>
             </item>
             <item row="6" column="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_buttons_power_long"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>RESET</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Short push:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Short push:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_buttons_power_short"/>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox_buttons_reset_short"/>
             </item>
             <item row="3" column="1">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>216</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Clock</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Watchdog</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_clock">
               <property name="text">
                <string>Enable Watchdog</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Clock</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_clock_sync_from_RTC">
               <property name="text">
                <string>Set system clock from RTC</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_clock_sync_from_system">
               <property name="text">
                <string>Set RTC from system clock</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_clock_manually">
               <property name="text">
                <string>Set system clock manually</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateTimeEdit" name="dateTimeEdit_clock"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>270</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Alarm</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Wakeup alarm settings</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_type_1">
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every day</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X hours</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X minutes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_type_2">
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every day</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X hours</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X minutes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_type_3">
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every day</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X hours</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X minutes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_type_4">
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every day</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X hours</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X minutes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_type_5">
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every day</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X hours</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Every X minutes</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Parameter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_parameter_1"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_parameter_2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_parameter_3"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_parameter_4"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_parameter_5"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Action</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_action_1">
               <item>
                <property name="text">
                 <string>Exec user script</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn on</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn off</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_action_2">
               <item>
                <property name="text">
                 <string>Exec user script</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn on</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn off</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_action_3">
               <item>
                <property name="text">
                 <string>Exec user script</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn on</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn off</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_action_4">
               <item>
                <property name="text">
                 <string>Exec user script</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn on</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn off</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_alarm_action_5">
               <item>
                <property name="text">
                 <string>Exec user script</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn on</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Turn off</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Script</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_script_1"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_script_2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_script_3"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_script_4"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_alarm_script_5"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_alarm_apply">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>IR</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Power Management</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
